Synapse/Api/Ragdoll.cs

```diff
@@ -25,6 +26,11 @@ public Ragdoll(RoleType roletype, Vector3 pos, Quaternion rot, Vector3 velocity,
 
         public string Name => ragdoll.name;
 
+        public RoleType RoleType
+        {
+            get => Server.Get.Host.ClassManager.Classes.FirstOrDefault(role => role.classColor == ragdoll.owner.ClassColor).roleId;
```

Comment:
How does the color of a class define what roletype a ragdoll is?
Also, isn't the classcolor the one from spectator, or whatever they are at the point _after_ dropping a ragdoll?